tests/integration/modules/molecule/event/playbook.yml

```diff
@@ -0,0 +1,210 @@
+---
+- name: Converge
+  collections:
+    - sensu.sensu_go
+  hosts: all
+  gather_facts: no
+  tasks:
+    - name: Create simple entity
+      entity:
+        auth:
+          url: http://localhost:8080
+        name: awesome_entity
+        entity_class: proxy
+      register: entity
+
+    - name: Create a simple check
+      check:
+        auth:
+          url: http://localhost:8080
+        name: awesome_check
+        command: /bin/true
+        output_metric_handlers:
+          - influxdb
+        subscriptions:
+          - checks
+          - also_checks
+        interval: 30
+      register: check
+
+    - name: Create event with minimal parameters
+      event:
+        auth:
+          url: http://localhost:8080
+        timestamp: 134532453
+        entity: "{{ entity.object }}"
+        check:
+          metadata:
+            name: not_so_awesome_check
+            namespace: default
+          interval: 60
+      register: result
+
+    - assert:
+        that:
+          - result is changed
+          - result.object.metadata.namespace == 'default'
+          - result.object.entity.metadata.name == 'awesome_entity'
+          - result.object.entity.entity_class == 'proxy'
+          - result.object.check.metadata.name == 'not_so_awesome_check'
+
+    - name: Create event with minimal parameters idempotence
+      event:
+        auth:
+          url: http://localhost:8080
+        timestamp: 134532453
+        entity: "{{ entity.object }}"
+        check:
+          metadata:
+            name: not_so_awesome_check
+            namespace: default
+          interval: 60
```

Comment:
@aljazkosir I was expecting following format:

```yaml
check: "{{ check.object }}"
check_merge:
  interval: 60
```

Wouldn't it be more user-friendly this way?